conf/reef/upgrades/upgrade_8-node-cluster_mix_os.yaml

```diff
@@ -2,7 +2,9 @@ globals:
   - ceph-cluster:
       name: ceph
       node1:
-        image-name: RHEL-8.6.0-x86_64-ga-latest
+        image-name:
+          openstack: RHEL-8.9.0-x86_64-ga-latest
+          ibmc: rhel-87-server-released
```

Comment:
IBM Cloud inventory needs to be changed, will share details.

Reply:
Okay, noted, once thats done, will send a PR to inculcate the changes.

Reply:
Use conf - `ibm-vpc-rhel-8.10-minimal-bx2-2x8`

Refer - https://github.com/red-hat-storage/cephci/pull/3756

Reply:
Done